lib/internal/Magento/Framework/HTTP/Client/Curl.php

```diff
@@ -229,6 +229,17 @@ public function get($uri)
         $this->makeRequest("GET", $uri);
     }
 
+    /**
+     * Make DELETE request
+     *
+     * @param string $uri uri relative to host, ex. "/index.php"
+     * @return void
+     */
+    public function delete($uri)
```

Comment:
Please add return type as void
```suggestion
    public function delete($uri): void
```

Please make this appropriate change in other methods as well.

Reply:
Pls also add parameters types definitions in all methods